addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -155,6 +155,11 @@ options.registry.newsletter_popup = options.registry.mailing_list_subscribe.exte
      * @override
      */
     start: function () {
```

Comment:
Commit message: "again on the newsletter popup again to see them again." -> 3x "again" in ~10words ^^ But this commit does not makes sense on its own: you introduce the "Display Thanks Button" option in the previous commit and add a third one to fix it :thinking: Just squash?